source/slides/19_high_availability.rst

```diff
@@ -0,0 +1,155 @@
+.. _19_high_availability:
+
+High Availability Strategies
+============================
+
+Terminology
+-----------
+
+* Redundancy
+* Reliability
+* Single point of failure (SPOF)
+* Horizontal Scaling
+* Vertical Scaling
+
+Redundancy
+----------
+
+Often measured as the number of nodes that can fail before a failure scenario
+occurs
+
+* Closely tied to reliability (more redundant sytems usually have higher
```

Comment:
You might as well remove this bullet and make it another paragraph. Looks a little funny otherwise.